docs(library-solidity): resolve variable shadowing in TestAsyncDecrypt example

qiwihui opened the pull request:
The `latestRequestId` variable was shadowed, causing `myCustomCallback` checks to fail.

qiwihui committed:
docs(decryption): resolve variable shadowing in TestAsyncDecrypt example

The `latestRequestId` variable was shadowed, causing `myCustomCallback` checks to fail.

qiwihui commented:
signed

aquint-zama commented:
@cla-bot check

aquint-zama commented:
Thanks for your PR @qiwihui, could you please rephrase your commit with `docs(library-solidity): …`?

qiwihui committed:
Merge branch 'zama-ai:main' into main

qiwihui committed:
docs(library-solidity): resolve variable shadowing in TestAsyncDecrypt example

The `latestRequestId` variable was shadowed, causing `myCustomCallback` checks to fail.

qiwihui committed:
Revert "docs(decryption): resolve variable shadowing in TestAsyncDecrypt example"

This reverts commit 9636c7d8d073a24a99118ac34443d6d32555714b.

qiwihui commented:
> Thanks for your PR @qiwihui, could you please rephrase your commit with `docs(library-solidity): …`?

Sure. A new commit has been committed and the older one has been revert. Please squash the commits when merging.

poppyseedDev commented:
minimal not worth merging, closing PR
